pyseir/load_data.py

```diff
@@ -303,15 +314,92 @@ def load_new_case_data_by_fips(fips, t0):
     observed_new_deaths: array(int)
         Array of new deaths observed each day.
     """
-
     _county_case_data = load_county_case_data()
     county_case_data = _county_case_data[_county_case_data['fips'] == fips]
     times_new = (county_case_data['date'] - t0).dt.days.iloc[1:]
     observed_new_cases = county_case_data['cases'].values[1:] - county_case_data['cases'].values[:-1]
     observed_new_deaths = county_case_data['deaths'].values[1:] - county_case_data['deaths'].values[:-1]
 
-    return times_new, observed_new_cases, observed_new_deaths
+    return times_new, observed_new_cases.clip(min=0), observed_new_deaths.clip(min=0)
+
+@lru_cache(maxsize=32)
+def load_hospitalization_data(fips, t0):
+    """
+    Obtain hospitalization data.
+
+    Parameters
+    ----------
+    fips: str
+        County fips to lookup.
+    t0: datetime
+        Datetime to offset by.
+
+    Returns
+    -------
+    times: array(float)
+        List of float days since t0 for the hospitalization data.
+    observed_hospitalizations: array(int)
+        Array of new cases observed each day.
+    type: str
+        'cumulative' or 'current'
+    """
+    hospitalization_data = CovidTrackingDataSource.local().timeseries()\
+        .get_subset(AggregationLevel.COUNTY, country='USA', fips=fips) \
+        .get_data(country='USA', fips=fips)
+
+    if len(hospitalization_data) == 0:
+        return None, None, None
+
+    times_new = (hospitalization_data['date'].dt.date - t0.date()).dt.days.values
+
+    if (hospitalization_data['current_hospitalized'] > 0).any():
+        return times_new, hospitalization_data['current_hospitalized'].values.clip(min=0), 'current'
+    elif (hospitalization_data['cumulative_hospitalized'] > 0).any():
+        return times_new, hospitalization_data['cumulative_hospitalized'].values.clip(min=0), 'cumulative'
+    else:
+        return None, None, None
```

Comment:
What is 'current'? Is it possible to pass a parameter to determine which category fo return?

Reply:
The problem is some states have current (will rename to occupancy) and some have cumulative. We don't know which until reading it.

Reply:
Added an enum